scripts/install_third_party_libs.py

```diff
@@ -79,6 +89,7 @@ def get_yarn_command():
     return 'yarn'
 
 
+# pylint: disable=redefined-outer-name
```

Comment:
Why do we need to disable lines below?

Reply:
Yeah, I had missed that variable, changed it now.
I am not sure but I think the error comes when I redefine a global variable within a function so I did that.

Reply:
But we aren't defining nor redefining any global variables though, so is it possible to remove?

Reply:
I looked up, it happens because a local name is identical to a global name([link](https://stackoverflow.com/questions/24999937/i-am-getting-the-error-redefined-outer-name)). Also removing it gives the same error.

Reply:
Which variable is it specifically? We should fix the root cause instead of disabling the check.

Reply:
Hi @kevinlee12 ! I am sorry I was traveling so could not address it early. I think the error comes if I use package/path/process or any other variable name anywhere in a function. I cant think of any other fix other than changing the names of these variables, can I change the variable names to global_{variable} (`package` to `global_package`)

Reply:
I’d recommend this instead: `package_name`, `target_path`. Let me if there are other variables.

Reply:
Yeah, the other variables are,
```
version
command
uextention
process
```
What do I rename them to? are these names fine?
```
version_number
command_text
uextention_text
current_process
```

Reply:
Yes, these are fine names.

Reply:
done!